mlrun/projects/project.py

```diff
@@ -2822,6 +2822,23 @@ def remove_function(self, name):
 
         :param name:    name of the function (under the project)
         """
+        # TODO: Remove this in 1.9.0
+        warnings.warn(
+            "'remove_function' is deprecated and will be removed in 1.9.0. "
+            "Please use `delete_function` instead.",
+            FutureWarning,
+        )
+        self.spec.remove_function(name)
```

Comment:
We keep deprecation notices for 2 versions, so this will be deleted in 1.10.0

Reply:
updated

Reply:
In the warning too, not just the comment :)

Reply:
And you can use the `@deprecated` wrapper